docs/docs/home.md

```diff
@@ -17,7 +17,7 @@ Explore our guides and examples to integrate Novu.
 
 ## What is Novu?
 
-Novu is an open-source notification infrastructure built for the engineering teams to help them build rich product notification experiences without constantly re-inventing the wheel.
+Novu is an open-source notification infrastructure built to help engineering teams build rich product notification experiences experiences without constantly re-inventing the wheel.
```

Comment:
```suggestion
Novu is an open-source notification infrastructure built to help engineering teams build rich product notification experiences without constantly re-inventing the wheel.
```

Reply:
Sorry, my bad! 
I'll be careful from now on.

Reply:
Don't worry about it! These things happen to everyone, thats why everyone gets code reviews ;)

Reply:
Thank you :)